observability/kiali/docinfo.xml

```diff
@@ -3,7 +3,7 @@
 <productnumber>3.0</productnumber>
 <subtitle>Using the Kiali Operator provided by Red Hat</subtitle>
 <abstract>
-    <para>This documentation provides information about the {KialiProduct}, and how to view the data that flows through your application.
+    <para>This documentation provides information about Kiali Operator provided by Red Hat, and how to view the data that flows through your application.
```

Comment:
I'm not sure if the issue could also exist on .xml files but consider the &nbsp variable between "Red" and "Hat":


https://redhat-documentation.github.io/supplementary-style-guide/#non-breaking-spaces